installer/README.md

```diff
@@ -29,3 +29,8 @@ ansible-playbook site.yml -i local.hosts
 ```bash
 ansible-playbook clean.yml -i local.hosts
 ```
+### Using Multi-cloud with dashboard
+* Install dashboard by following the dashboard installation guide [here](https://github.com/anvithks/opensds-dashboard/blob/dashboard-ansible-installer/install/readme.md).
```

Comment:
Can you please remove/update personal link for documentation

Reply:
done.